sphinx/conf.py

```diff
@@ -118,8 +118,10 @@
     ("py:attr", "django.contrib.auth.models.User.feedback"),
     ("py:attr", "django.contrib.auth.models.User.mfa_keys"),
     ("py:attr", "django.contrib.auth.models.User.imprint_translations"),
-    ("py:attr", "django.contrib.auth.models.User.page_translations"),
+    ("py:class", "django.utils.datastructures.MultiValueDict"),
+    ("py:class", "django.contrib.auth.models.User.page_translations"),
```

Comment:
```suggestion
```
The `class`-line can be removed since you added the correct `attr` two lines after